lib/backend/src/Obelisk/Backend.hs

```diff
@@ -171,5 +171,5 @@ renderGhcjsFrontend urlEnc route f = do
       ghcjsPreload = elAttr "link" ("rel" =: "preload" <> "as" =: "script" <> "href" =: "ghcjs/all.js") blank
       ghcjsScript = elAttr "script" ("language" =: "javascript" <> "src" =: "ghcjs/all.js" <> "defer" =: "defer") blank
   liftIO $ renderFrontendHtml urlEnc route
-    (_frontend_head f >> injectExecutableConfigs >> baseTag >> ghcjsPreload)
+    (baseTag >> _frontend_head f >> injectExecutableConfigs >> ghcjsPreload)
```

Comment:
Huh, I wouldn't have expected the order to matter here; can you elaborate, and maybe add a comment mentioning that it's important?

Reply:
Will do and I will test again to make really sure - but I tested twice already, so I am pretty sure already.